src/framework/parsers/draco-decoder.js

```diff
@@ -0,0 +1,168 @@
+import { WasmModule } from "../../core/wasm-module";
+import { ABSOLUTE_URL } from "../asset/constants";
+import { DracoWorker } from "./draco-worker";
+
+class JobQueue {
```

Comment:
It'd be great to have this documented.
Also, it this a generic class, or is it draco specific?

Reply:
This job queue is currently only used for draco, but I think i'd like to use it for basis if possible. Will probably make sense to document then.